### PR TITLE
WIP: Store dracut kickstart comands in boss

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Mostly trying to see which parts would work. The code probably needs a different layout, maybe a child module, not really sure yet.

### CODE REVIEW

File: pyanaconda/modules/boss/kickstart_manager/kickstart_manager.py [hunk 1]
```diff
@@ -157,6 +230,18 @@ def _generate_from_modules(self):
...
+    def _generate_from_direct(self):
+        """Generate kickstart from the direct observer.
+
+        :return: kickstart strings
+        """
+        if not self._direct_observer:
+            return ""
+
+        super_instance = super(type(self._direct_observer), self._direct_observer)
+        kickstart = super_instance.generate_kickstart()
```
Comment (VladimirSlavik) [inline pyanaconda/modules/boss/kickstart_manager/kickstart_manager.py:242]: This in particular is just wrong, don't do this :-) It doesn't really work.

**Comment by VladimirSlavik** [comment]
This now works, but the actual organization of the code is still unfinished.

**Review by jkonecny12 — COMMENTED** [review]
Looks good to me so far.

**Comment by VladimirSlavik** [comment]
Still needs additional tests for `KickstartManager._distribute_to_direct()`, and splitting more things from the main commit.

I am not sure if this is fine or if there should be another layer of abstraction to mate the ks manager with both the dbus and direct modules. My plan is to finish the above, completing the approach without this abstraction layer, and try the other form as further changes on top of this.

**Comment by VladimirSlavik** [comment]
The last two commits are a rough outline of how "unified observers" might look in terms of code.

**Comment by VladimirSlavik** [comment]
(Now it also works.)

**Pull request closed without merging**